cmd/livepeer/livepeer.go

```diff
@@ -134,6 +134,7 @@ func main() {
 	ethKeystorePath := flag.String("ethKeystorePath", "", "Path for the Eth Key")
 	ethOrchAddr := flag.String("ethOrchAddr", "", "ETH address of an on-chain registered orchestrator")
 	ethUrl := flag.String("ethUrl", "", "Ethereum node JSON-RPC URL")
+	replaceTx := flag.Bool("replaceTransactions", true, "Set to true to replace ethereum transactions with a higher gas price if they time out")
```

Comment:
I think this can be a neat addition yeah. One thing that I don't really like here is the hardcoded `EThTxTimeout`, while a user can set the number of max replacements to do it is not immeadiatly known how often the replacement will be tried. 

Allowing a `ethTxTimeout` flag could help here but also again increases the amount of config params required. 

What are your thoughts on that?

Reply:
Oops missed this comment in the first couple of reviews.

I think it makes sense to allow the timeout to be configurable. What about `-transactionTimeout`?